debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/RecordMakersTest.java

```diff
@@ -161,6 +164,40 @@ public void shouldGenerateRecordForDeleteEvent() throws InterruptedException {
         assertThat(tombstone.valueSchema()).isNull();
     }
 
+    @Test
```

Comment:
Could you add the `@FixFor()` annotation?

Reply:
No probs, will do, once I create a JIRA for it

Reply:
That's done